api/controllers/mobile/manifest.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * mobile/manifest.js
+ * @apiDescription Respond with the manifest.json of the Mobile PWA
+ *
+ * @api {get} /mobile/app/:tenantID/:ID/manifest.json
+ * @apiGroup Mobile
+ * @apiPermission None
+ * @apiUse successRes
+ * @apiSuccess (200) {html}
+ */
+// const authLogger = require("../../lib/authLogger.js");
+
+// var inputParams = {
+//    tenant: { string: true, optional: true },
+// };
+
+module.exports = function (req, res) {
+   req.ab.log("mobile/manifest():");
+
+   let tenantID = req.ab.param("tenantID");
```

Comment:
Why not use the tenant id from the `authTenant` policy? If we're not using maybe we should skip `authtTenant` for this route?

Reply:
mostly 'cause the route required the `:tenantID` :  `{get} /mobile/app/:tenantID/:ID/manifest.json`

But it does make more sense to pull the authenticated ID from the `authTenant` policy.